src/util/types.ts

```diff
@@ -1193,6 +1193,8 @@ export interface LabelLayoutOption {
         | 'shiftY'
         | 'shuffleX'
         | 'shuffleY'
+        | 'repel'
```

Comment:
Is `repel` the name of the algorithm?  What does it mean?

From the code I guess it's like force directed layout.